contracts/p1/StRSR.sol

```diff
@@ -416,6 +418,8 @@ abstract contract StRSRP1 is Initializable, ComponentP1, IStRSR, EIP712Upgradeab
         // Transfer RSR to caller
         emit ExchangeRateSet(initRate, exchangeRate());
         IERC20Upgradeable(address(rsr)).safeTransfer(_msgSender(), seizedRSR);
+
+        rsrRewardsAtLastPayout = rsrRewards();
```

Comment:
This breaks checks-effects-interactions...not sure what to do about it though. 

`rsrRewardsAtLastPayout = rsrRewards() - seizedRSR;` doesn't seem _great_

Reply:
Yeah, that was the previous solution, but ended up with this since it most closely follows separation of concerns while mildly breaking checks-effects-interactions.